enterprise_access/apps/api/v1/tests/test_subsidy_access_policy_views.py

```diff
@@ -1288,6 +1297,11 @@ def test_credits_available_endpoint(
             enterprise_customer_uuid=self.enterprise_uuid,
             active=False,
         )
+        # The following policy should never be returned as it has redeemability disabled.
+        PerLearnerEnrollmentCapLearnerCreditAccessPolicyFactory(
+            enterprise_customer_uuid=self.enterprise_uuid,
+            redeemability_disabled_at=datetime.utcnow() - timedelta(days=1),
+        )
```

Comment:
As discussed at standup on Friday, when redeemability is disabled (was: policy is "retired"), we should make the policy effectively disappear from a learner's perspective.  I.e. it should not be returned by the credits_available endpoint, tested here.